packages/astro/src/@types/astro.ts

```diff
@@ -462,6 +462,24 @@ export interface AstroUserConfig {
 	 */
 	site?: string;
 
+
+	/**
+	 * @docs
+	 * @name compressHTML
+	 * @type {boolean}
+	 * @default `false`
+	 * @description
+	 * When enabled Astro removes all whitespace from your final HTML, including line breaks. This is useful for reducing the size of HTML transfered over the wire.
+	 * To enable this, set the `compressHTML` flag to `true`.
```

Comment:
From docs, we should explain that this feature is enabled in both development and production.